### PR TITLE
snapcraft 7.1.4

### PR DESCRIPTION
**Pull request by singingwolfboy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Upgrading snapcraft to the latest version.

### CODE REVIEW

File: Formula/snapcraft.rb [hunk 1]
```diff
@@ -4,31 +4,20 @@ class Snapcraft < Formula
...
-  bottle do
-    rebuild 1
-    sha256 cellar: :any,                 arm64_monterey: "4dddbb0763167c0dd55efbf9de703d98aef64a33832a48bbd11ba67ff976dce0"
-    sha256 cellar: :any,                 arm64_big_sur:  "4f694957b000eeac82f0bd412e8e35f7343f163fb0be7d3d75b31748bf2621aa"
-    sha256 cellar: :any,                 monterey:       "fa6f1500f39010225c398634ce3a8b80c81d43e344deb7921912c4c97b4ef767"
-    sha256 cellar: :any,                 big_sur:        "73e7d77afaf667484b821c6c43560461cd484f9ea031f603084a942aa1bdadf3"
-    sha256 cellar: :any,                 catalina:       "aeeabe599366fce9157670a566b6624f3a9b78e5078287632a4ab766120fd379"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "fcd639e34e8e1a0c183938cd95d2663a343d99c6ff981bbaad5d16873b11ab03"
-  end
-
```
Comment (chenrui333) [inline Formula/snapcraft.rb:26]: no need to touch `bottle` block, it would be managed by @BrewTestBot

File: Formula/snapcraft.rb [hunk 2]
```diff
@@ -283,63 +317,68 @@ class Snapcraft < Formula
...
-    url "https://files.pythonhosted.org/packages/ae/3d/9d7576d94007eaf3bb685acbaaec66ff4cdeb0b18f1bf1f17edbeebffb0a/tabulate-0.8.9.tar.gz"
-    sha256 "eb1d13f25760052e8931f2ef80aaf6045a6cceb47514db8beab24cded16f13a7"
-  end
-
-  resource "testresources" do
-    url "https://files.pythonhosted.org/packages/9d/57/8e3986cd95a80dd23195f599befa023eb85d031d2d870c47124fa5ccbf06/testresources-2.0.1.tar.gz"
-    sha256 "ee9d1982154a1e212d4e4bac6b610800bfb558e4fb853572a827bc14a96e4417"
+    url "https://files.pythonhosted.org/packages/7a/53/afac341569b3fd558bf2b5428e925e2eb8753ad9627c1f9188104c6e0c4a/tabulate-0.8.10.tar.gz"
+    sha256 "6c57f3f3dd7ac2782770155f3adb2db0b1a269637e42f27599925e64b114f519"
   end
```
Comment (chenrui333) [inline Formula/snapcraft.rb:347]: this can be `depends_on "libpython-tabulate"`

**Comment by singingwolfboy** [comment]
@chenrui333 I believe I fixed the problems you pointed out. Could I get another review, please?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @singingwolfboy!

**Pull request closed without merging**